examples/generator-full-stack-example/graphback.json

```diff
@@ -18,10 +18,10 @@
   },
   "folders": {
     "model": "./model",
-    "resolvers": "./src/resolvers/",
-    "schema": "./src/schema",
+    "resolvers": "./server/src/resolvers/",
```

Comment:
Is this file needed now that we have `.graphqlrc`?

Reply:
Good point! It will be needed until we totally remove the CLI from graphback. 
I think we going to keep it for the time being in deprecated state while still allowing people to use it. There will be no templates etc. This is more to discussion on how we polish the release. In this PR I wanted to enable GraphQL Cli but do not close dors on our own cli still used in tests etc.